test/TestXlsx.hs

```diff
@@ -439,17 +433,18 @@ testFormatWorkbookResult :: Xlsx
 testFormatWorkbookResult = def & xlSheets .~ sheets
                                & xlStyles .~ renderStyleSheet style
   where
-    testCellMap1 = M.fromList [((1, 1), Cell { _cellStyle   = Nothing
-                                             , _cellValue   = Just (CellText "text at A1 Sheet1")
-                                             , _cellComment = Nothing
-                                             , _cellFormula = Nothing })]
-    testCellMap2 = M.fromList [((2, 3), Cell { _cellStyle   = Just 1
-                                             , _cellValue   = Just (CellDouble 1.23456)
-                                             , _cellComment = Nothing
-                                             , _cellFormula = Nothing })]
-    sheets = [ ("Sheet1", def & wsCells .~ testCellMap1)
-             , ("Sheet2", def & wsCells .~ testCellMap2)
-             ]
+    testCellMap1_ = M.fromList [((1, 1), Cell { _cellStyle   = Nothing
```

Comment:
where did these trailing underscores come from?

Reply:
it's a way to sidestep name shadowing by this binding

Reply:
but where does this shadowing come from?

Reply:
Top level bindings:

`testCellMap1` line 143, `testCellMap2` L. 183

Reply:
ouch, those are actually seem to be only used in `testXlsx`, what do you think about moving them into `where` as a proper way to fix shadowing?

Reply:
Actually `testCellMap1` is also used in Main.

I've renamed the local bindings of `testFormatWorkbookResult` to solve the issue